CONTRIBUTING.md

```diff
@@ -102,6 +102,13 @@ Exceptionally, commit type `docs` may omit scope. e.g, docs: update contracts/RE
 If the change affects more than one scope, the commit scope may be omitted.
 e.g, feat: add Colosseum contract
 
+### Commit Rule
+
+There are the following rules for commit messages:
+
+- The commit title should always start with an imperative verb.
```

Comment:
How about adding the rules for the Pull Requests as well in a similar manner? There might be other more rules for Pull Requests, but at least we need a rule with the PR title.

Reply:
I think a "Commit Rules" category would be nice, though parts of that would overlap with the PR rule -> aka like "If the change affects more than one scope, the commit scope may be omitted. e.g, feat: add Colosseum contract" -> so so thinking of what goes where might be needed.

Reply:
Since this PR is pending so long, I think I want to ask @ashjeong to continue on it.